docs/ota-client-guide/modules/ROOT/pages/client-provisioning-methods.adoc

```diff
@@ -81,10 +81,11 @@ image::img::device-cred-provisioning.png[width=100%]
 If you want to use "shared credential" provisioning, you don't have to do anything at all. When your account was created, we already generated a Fleet Root CA and keypair for you, and stored them on the OTA Connect server. We take the security of these keys extremely seriously: following industry best practices, they are kept in a Vault instance and only taken out when you request them.
 
 If you want to use "device credential" provisioning, you'll need to provide us with your own Fleet Root CA so that the OTA Connect server can verify devices.
-Of course, you can use both methods, but in that case, we recommend that you maintain separate user accounts:
+Of course, you can use both methods, but in that case, we recommend that you maintain separate xref:ota-web::environments-intro.adoc[environments]:
 
-* one account for testing with "shared credential" provisioning
-* one account for production with "device credential" provisioning
+* one environment for testing with "shared credential" provisioning
+* one environment for production with "device credential" provisioning
+
+It is not possible to migrate from a test environment to a production environment. Therefore we recommend that you test with devices that will not go into production or devices that can be completely wiped and reset once they are ready to deploy.
```

Comment:
I've added the comma, but I think `will` best describes the scenario here better because changing it implies all users are certain of their devices before testing, which might not always be the case.

Reply:
The sentence itself implies that users should use a certain type of devices for testing. Also, we provide users with a rule of what is better to use. Therefore, I would rather go for present here :)